phpBB/language/en/composer.json

```diff
@@ -0,0 +1,27 @@
+{
+	"name": "en",
```

Comment:
Always need a vendor name. `phpbb/en-language-pack` Naming here doesn't really matter (and I don't think that requiring specific naming conventions would be a good idea to be honest).